src-gui/src/renderer/components/modal/seed-selection/SeedSelectionDialog.tsx

```diff
@@ -36,6 +36,7 @@ export default function SeedSelectionDialog() {
     SeedChoice["type"] | undefined
   >("RandomSeed");
   const [customSeed, setCustomSeed] = useState<string>("");
+  const [blockheightInput, setBlockheight] = useState<string | undefined>();
```

Comment:
Rename `setBlockheight` to `setBlockheightInput` to make it clear what state we are modifying when calling this function.